manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -197,21 +193,21 @@ spec:
           value: "{{ $value }}"
         {{- end }}
         {{- end }}
-        {{- if .Values.ztunnel.env }}
-        {{- range $key, $val := .Values.ztunnel.env }}
+        {{- with .Values.env }}
+        {{- range $key, $val := . }}
         - name: {{ $key }}
           value: "{{ $val }}"
         {{- end }}
         {{- end }}
       containers:
       - name: istio-proxy
-{{- if contains "/" .Values.ztunnel.image }}
-        image: "{{ .Values.ztunnel.image }}"
+{{- if contains "/" .Values.image }}
+        image: "{{ .Values.image }}"
 {{- else }}
-        image: "{{ .Values.ztunnel.hub | default .Values.global.hub }}/{{ .Values.ztunnel.image | default "proxyv2" }}:{{ .Values.ztunnel.tag | default .Values.global.tag }}{{with (.Values.ztunnel.variant | default .Values.global.variant)}}-{{.}}{{end}}"
+        image: "{{ .Values.hub }}/{{ .Values.image | default "proxyv2" }}:{{ .Values.tag }}{{with (.Values.variant )}}-{{.}}{{end}}"
```

Comment:
Any reason to have the default be proxyv2 for ztunnel here? the before had proxyv2, just curious why we would default to proxyv2 instead of ztunnel.